src/components/Bird/Aithyia.js

```diff
@@ -122,14 +122,14 @@ const renderFirstEdition = () => (
           , 89
           </a>
           {' '}
-          names it with
+          names it with 
```

Comment:
Due to the way that whitespace is handled in JSX (the dialect of XML used by React), adding spacing this way won't show up in the HTML unfortunately. The way to get the space to show up is to add `{' '}` on the line below instead of adding a space at the end of the line:


```
names it with
{' '}
```